SimplenoteUITests/AllNotes.swift

```diff
@@ -61,26 +61,49 @@ class AllNotes {
         for _ in 0..<notesNumber {
             let cell = app.tables.cells.element(boundBy: startingIndex)
             cell.swipeLeft()
-            cell.buttons[UID.Button.NoteCellTrash].tap()
+            cell.buttons[UID.Button.noteCellTrash].tap()
         }
     }
 
     class func waitForLoad() {
-        let allNotesNavBar = app.navigationBars[UID.NavBar.AllNotes]
+        let allNotesNavBar = app.navigationBars[UID.NavBar.allNotes]
         let predicate = NSPredicate { _, _ in
-            allNotesNavBar.staticTexts[UID.Text.AllNotesInProgress].exists == false
+            allNotesNavBar.staticTexts[UID.Text.allNotesInProgress].exists == false
         }
 
         let expectation = XCTNSPredicateExpectation(predicate: predicate, object: .none)
         XCTWaiter().wait(for: [expectation], timeout: 10)
     }
+
+	class func searchForText(text: String) {
+		print(">>> Searching for '" + text + "'")
+		let searchField = app.searchFields[UID.SearchField.search]
+		guard searchField.exists else { return }
+		searchField.tap()
+
+		let clearTextButton = searchField.buttons[UID.Button.clearText]
+		if clearTextButton.exists {
+			clearTextButton.tap()
+		}
+
+		searchField.typeText(text)
+		sleep(2)
+	}
+
+	class func searchCancel() {
+		print(">>> Canceling Search")
+		let searchCancelButton = app.buttons[UID.Button.cancel]
+		guard searchCancelButton.exists else { return }
+		searchCancelButton.tap()
+	}
 }
 
 class AllNotesAssert {
 
     class func noteExists(noteName: String) {
-        print(">>> Asserting note existsence: " + noteName)
-        XCTAssertTrue(app.tables.cells[noteName].exists, "\"" + noteName + noteNotFoundInAllNotes)
+        print(">>> Asserting that note is shown once: " + noteName)
+		let matchesCount = Table.getCellsWithExactLabelCount(label: noteName)
+		XCTAssertEqual(1, matchesCount)
```

Comment:
```suggestion
		XCTAssertEqual(matchesCount, 1)
```

The XCTest convention is to have the actual value as the first parameter and the expected one as the second.

See the [example in the `XCTAssertEqual` documentation](https://developer.apple.com/documentation/xctest/2142776-xctassertequal) or the one in the [XCTest introduction](https://developer.apple.com/documentation/xctest/defining_test_cases_and_test_methods).

<table>
<tr>
	<td>

![image](https://user-images.githubusercontent.com/1218433/109105249-a3f28a80-7781-11eb-810f-3c4afabf2f78.png)

</td>
	<td>

![image](https://user-images.githubusercontent.com/1218433/109105273-b371d380-7781-11eb-9073-274de53db43c.png)

</td>

</table>

Reply:
Thanks Gio! Definitely my miss, it was the other way around in the framework I used previously. Changed this everywhere.